app/views/papers/_sort_by.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="btn-group btn-group-xs nav-button-group sort" role="group" aria-label="...">
+  <%= link_to 'Votes', page == 'list' ? list_path(sort: 'votes') : papers_path(sort: 'votes'), class: "btn btn-default" + (params[:sort] == "pub_date" ? "" : " active"), role: "button" %>
+  <%= link_to 'Publish Date', page == 'list' ? list_path(sort: 'pub_date') : papers_path(sort: 'pub_date'), class: "btn btn-default" + (params[:sort] == "pub_date" ? " active" : ""), role: "button"  %>
```

Comment:
Check docs to see if you can, I mean.


This is also a code smell: `class: "btn btn-default" + (params[:sort] == "pub_date" ? "" : " active")`

I would put the additional space at the end of `btn-default` instead of putting it in your added classes. You are also using ternary operators for essentially no reason. If the other option is "", then you can basically just use an if statement. So `(" active" if params[:sort] == "pub_date")` and `(" active" if params[:sort] == "votes")`. While we're at it, you should just use string interpolation for the classes: `class: "btn btn-default #{'active' if params[:sort] == "votes"}"`

I still feel like there's something weird about having that logic in there, but right now that's a simple way to do it.